src/app/commands/cmd_change_pixel_format.cpp

```diff
@@ -475,7 +477,8 @@ void ChangePixelFormatCommand::onExecute(Context* context)
           SelectedLayers selLayers;
           for (auto layer : sprite->root()->layers())
             selLayers.insert(layer);
-          job.tx()(new cmd::FlattenLayers(sprite, selLayers));
+          job.tx()(new cmd::FlattenLayers(sprite, selLayers,
+                                          Preferences::instance().experimental.newBlend()));
```

Comment:
Same here as in the `cmd_change_pixel_format.cpp`, this lambda is executed in a background thread, I'm not even sure that the code is already thread-safe, but anyway, just in case we can move the `newBlend` outside. You could replace it with something like:
```c++
const bool newBlend = Preferences::instance().experimental.newBlend();
...
job.startJobWithCallback(
  [this, &job, flatten, newBlend] {
    ...
    job.tx()(new cmd::FlattenLayers(sprite, selLayers, newBlend));
...
```